src/acrtransfer/azext_acrtransfer/commands.py

```diff
@@ -0,0 +1,47 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+
+from azure.cli.core.commands import CliCommandType
+from azext_acrtransfer._client_factory import cf_acrtransfer
+from ._format import import_pipeline_output_format, export_pipeline_output_format, pipeline_run_output_format
+
+
+def load_command_table(self, _):
+    importpipeline_sdk = CliCommandType(
+        operations_tmpl='azext_acrtransfer.vendored_sdks.containerregistry.v2019_12_01_preview.operations#ImportPipelinesOperations.{}',
+        client_factory=cf_acrtransfer,
+        min_api='2019-12-01-preview'
+    )
+
+    exportpipeline_sdk = CliCommandType(
+        operations_tmpl='azext_acrtransfer.vendored_sdks.containerregistry.v2019_12_01_preview.operations#ExportPipelinesOperations.{}',
+        client_factory=cf_acrtransfer,
+        min_api='2019-12-01-preview'
+    )
+
+    pipelinerun_sdk = CliCommandType(
+        operations_tmpl='azext_acrtransfer.vendored_sdks.containerregistry.v2019_12_01_preview.operations#PipelineRunsOperations.{}',
+        client_factory=cf_acrtransfer,
+        min_api='2019-12-01-preview'
+    )
+
+    with self.command_group('acr import-pipeline', importpipeline_sdk, table_transformer=import_pipeline_output_format, is_preview=True) as g:
```

Comment:
Can you use a subgroup `acr pipeline` and then you have `import/export/run` commands under this subgroup?

Reply:
I could, although I feel like it's more user-friendly as it is. Would it be okay to leave it the way it is or should we definitely change it?

Reply:
I realized `import-pipeline` and `export-pipeline` are 2 types of pipelines, it's OK to leave it as it is. It's just that the pipeline subgroups (`import-pipeline`, `export-pipeline`, `pipeline-run`) will not be shown together in alphabetic order when users run `az acr -h`.

Reply:
Gotcha. Yeah that's fine with me, I'd like to keep it as-is